docs/android-feature-api.md

```diff
@@ -6,27 +6,34 @@ slug: /android-feature-api
 
 ## Manifest
 
-This file should live in Fenix and be readable by the Android Feature API:
+This file should live in Fenix and be readable by the Android Feature API.
+
+Note that:
+
+- Default values are required for Android/iOS
+- Valid `type`s are `boolean`, `string`, `int`, `drawable`
+- `choices` is optional
 
 ```yaml
 default_menu_message:
   description: "A menu message that asks people to set to default",
```

Comment:
I ran this doc through a YAML-to-JSON converter and it didn't like the trailing commas. Not sure if that is valid in other parsers.